snap/README.md

```diff
@@ -365,164 +377,177 @@ curl -v --cacert /var/snap/edgeca/current/CA.pem -X GET https://server01:8443/co
+| smtp.port                   | SMTP port|
+| smtp.sender                 | Notification message sender|
+| smtp.enable-self-signed-cert| Enable support for invalid (self-signed) certificates|
+| smtp.subject                | Notification message subject|
+| smtp.auth-mode              | User need to store the credential via the /secret API before sending the email notification. AuthMode is the SMTP authentication mechanism. Currently, `usernamepassword' is the only AuthMode supported by this service, and the secret keys are 'username' and 'password'. |
 
 ## Building
 
-The snap is built with [snapcraft](https://snapcraft.io), and the snapcraft.yaml recipe is located within `edgex-go`, so the first step for all build methods involves cloning this repository:
+The snap is built with [snapcraft](https://snapcraft.io), and the snapcraft.yaml recipe is located within `edgex-go`, 
+so the first step for all build methods involves cloning this repository:
 
 ```bash
-$ git clone https://github.com/edgexfoundry/edgex-go
-$ cd edgex-go
+git clone https://github.com/edgexfoundry/edgex-go
+cd edgex-go
 ```
 
 ### Installing snapcraft
 
-There are a few different ways to install snapcraft and use it, depending on what OS you are building on. However after building, the snap can only be run on a Linux machine (either a VM or natively). To install snapcraft on a Linux distro, first [install support for snaps](https://snapcraft.io/docs/installing-snapd), then install snapcraft as a snap with:
+There are a few different ways to install snapcraft and use it, depending on what OS you are building on. However after building, 
+the snap can only be run on a Linux machine (either a VM or natively). To install snapcraft on a Linux distro, 
+first [install support for snaps](https://snapcraft.io/docs/installing-snapd), then install snapcraft as a snap with:
 
 ```bash
-$ sudo snap install snapcraft
+sudo snap install snapcraft
 ```
 
 (note you will be promted to acknowledge you are installing a classic snap - use the `--classic` flag to acknowledge this)
 
-**Note** - currently the snap doesn't support cross-compilation, and must be built natively on the target architecture. Specifically, to support cross-compilation the kong/lua parts must be modified to support cross-compilation. The openresty part uses non-standard flags for handling cross-compiling so all the flags would have to manually passed to build that part. Also luarocks doesn't seem to easily support cross-compilation, so that would need to be figured out as well.
+**Note** - currently the snap doesn't support cross-compilation, and must be built natively on the target architecture. 
+Specifically, to support cross-compilation, the kong part must be modified:
+* OpenResty uses non-standard flags for handling cross-compiling so all the flags would have to manually passed to build that part.
+* LuaRocks doesn't seem to easily support cross-compilation and needs further investigation.
```

Comment:
This part is probably no longer valid - as we don't build Kong anymore (as per https://github.com/edgexfoundry/edgex-go/pull/3816 ) but install it from a pre-built debian archive. So I think this paragraph about modifying Kong, OpenResty and LuaRocks should just be removed.

Reply:
Deleted. Thanks!